### DISCUSSION TITLE
Sistema automazioni luci e sistema allarme - Base

### DISCUSSION THREAD

**Discussion by Ilosdea** [indomus/forum#3117] [GENERALE - Discussioni generiche]
Buongiorno a tutti!
Prima di tutto mi presento, sono Davide e non ho molta esperienza con sistemi di automazione e mi sto formando (anche se per ora in maniera teorica) tramite le vostre guide per le quali, come già fatto da molti, vi faccio i complimenti e ringrazio.
Avrei due domande (molto teoriche lo so e per le quali in realtà chiedo solo se si possa fare o meno, perché la mia idea è farlo, in maniera molto graduale dedicandoci maggiore tempo tra qualche settimana che sarò completamente in ferie).
Sto facendo dei lavori a casa e vorrei implementare (ma in un secondo momento) due sistemi di automazione.

Automazioni da voler implementare: 
- Sistema illuminazione corridoi
Fare in modo che da una certa ora, in base alla presenza e movimento di persone all'interno di un corridoio si accendano due strisce led (che sono tra loro indipendenti) con una determinata tonalità meno forte, mentre da una certa ora in poi si accendano, in base sempre alla presenza o movimento di persone all'interno dei corridoi, con una luminosità maggiore.
Il tutto però che possa essere non solo automatizzato ma che possa anche essere gestito con i semplici interruttori di accensione spegnimento.
Come sensore presenza pensavo all'aqara FP2, come strisce led nessuna idea in particolare.

- Sistema allarme
Utilizzare un sistema di allarme (idealmente Amazon ring XL) che sia composto da 3/4 sensori di apertura finestra, più 2 sensori apertura porta, più almeno due telecamere da interno.
Le due porte di ingresso si trovano una al piano di sopra (con apertura con la classica chiave) l'altra al piano di sotto (con apertura con Nuki Smart Pro 3 più keypad).
L'idea è avere il controllo dell'allarme con tastierino di attivazione/disattivazione nel piano di sopra mentre fare in modo che si disattivi automaticamente se entro nel piano di sotto utilizzando il codice o l'impronta con Nuki, senza dover andare sopra per disattivare l'allarme (o senza doverlo fare con il telefono visto che mi può capitare di non avere il telefono con me).
Le due aree della casa sono tra loro distanti (ma soprattutto con dei muri spessi a separarle, visto che parliamo di una costruzione del 1500) l'idea è utilizzare un ripetitore wifi che possa amplificare la portata del wifi anche nella zona di sotto.

Messaggio lunghissimo lo so, se qualcuno ha voglia e pazienza di rispondere grazie mille! 
È possibile implementare queste due mie idee?  (ovviamente eventualmente abbiate suggerimenti grazie mille)!

**Comment by KC-inDomus** [comment]
prima di entrare nel dettaglio, la domanda principale è: quale HUB personale hai intenzione di adottare?

**Reply by Ilosdea** [reply]
Scusami, lo davo per scontato home assistant, l'idea è seguire le raccomandazioni che date nel sito installandolo su un mini pc (prima provando la versione raccomandata core su docker e, in caso non riesca, la versione OS)

**Reply by KC-inDomus** [reply]
diciamo che alla lue di quanto hai a grandi linee descritto è tutto più che fattibile. sono alla fine automazioni piuttosto semplici, te ne renderai conto da te mano mano che le metterai in campo. con home assistant puoi fare molto, molto di più. sull'hardware che hai citato personalmente mi trovo, per le striscie led valuta shelly rgbw2 oppure qualche kit completo, purché sia integrabile.

**Reply by Ilosdea** [reply]
Perfetto ti ringrazio! Man mano proverò a fare i vari passaggi! Intanto grazie mille del supporto e del consiglio (ovviamente anche se hai idea di un kit completo integrabile segnalamelo pure).
Intanto grazie mille ancora!
